cl/lib/email_backends.py

```diff
@@ -161,7 +161,6 @@ def send_messages(
         r = make_redis_interface("CACHE")
         msg_count = 0
         for email_message in email_messages:
-            message = email_message.message()
```

Comment:
`.message()` does not modify anything in place and `message` isn't used anywhere.